app/jobs/story_update_job.rb

```diff
@@ -20,7 +20,7 @@ def receive_story_update(data)
     load_resources(data)
     episode ? update_episode : create_episode
```

Comment:
And hey, just saw a `undefined method 'copy_media' for nil:NilClass` from feeder-staging.  Progress!